### ISSUE TITLE
label `asn` too short integer 

### ISSUE THREAD

**Issue by xi-yang** [fabric-testbed/InformationModel#133]
I am getting exception while trying to set FABRCI ASN `398900` to the `asn` label, which appears defined as a short integer. 
```
fim.slivers.capacities_labels.LabelException: Label exception: Provided label value 398900 for asn must be in a valid range 1-65535
```

**Comment by ibaldin** [comment]
Whaaa! Who new ASNs could be this large. I will create a fix shortly.

**Comment by ibaldin** [comment]
Addressed in #132 in FIM-1.3.1
